mkdocs.yml

```diff
@@ -23,7 +23,7 @@ nav:
 plugins:
   - search
   - mkdocs-jupyter:
-      execute: False
+      execute: True
```

Comment:
The getting started notebook (not in this repo but in another PR) is slightly tricky. It will need `graphviz` to render correctly which might not exist on github runners. I wonder if you could add an exclusion rule to that notebook (`docs/notebooks/getting_started.ipynb`)? I think `mkdocs-jupyter` supports that

Reply:
the `graphviz` issue might come up in other notebooks too? 
If this applies also to `main_tutorial.ipnb` e.g., `graphviz` is used in there for sure.